sklearn/feature_extraction/image.py

```diff
@@ -305,47 +305,6 @@ def _extract_patches(arr, patch_shape=8, extraction_step=1):
     return patches
 
 
-@deprecated("The function feature_extraction.image.extract_patches has been "
```

Comment:
OMG I didn't remember this! Wait so we deprecated the n-d version but not the 2d version? hm